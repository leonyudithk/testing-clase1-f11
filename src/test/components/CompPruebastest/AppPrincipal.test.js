import { shallow } from 'enzyme'
import React from 'react'
import AppPrincipal from '../../../components/CompPruebasTest/AppPrincipal'

describe('Prueba al Componente AppPrincipal', ()=>{

    test('validar el renderizado en AppPrincipal', ()=>{

        //mesa trabajo - Arrglear
        const saludo1 = 'Bienvenidos Frontend 11'
        const subtitulo1 = 'Les quedas 4 clases para finalizar'
        const wrapper = shallow(<AppPrincipal saludo={saludo1} subtitulo={subtitulo1}/>)
       
        // ACtuar
        expect(wrapper).toMatchSnapshot()

    })

})