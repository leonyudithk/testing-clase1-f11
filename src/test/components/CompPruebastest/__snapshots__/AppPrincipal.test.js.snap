// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba al Componente AppPrincipal validar el renderizado en AppPrincipal 1`] = `
<div>
  <h1>
    Bienvenidos Frontend 11
  </h1>
  <h3>
    Les quedas 4 clases para finalizar
  </h3>
  <p>
    La informaci√≥n esta correcta
  </p>
</div>
`;
